fdbclient/MultiVersionTransaction.actor.cpp

```
@@ -2453,6 +2453,12 @@ void MultiVersionApi::setNetworkOptionInternal(FDBNetworkOptions::Option option,
 		externalClient = true;
 		bypassMultiClientApi = true;
 		forwardOption = true;
+	} else if (option == FDBNetworkOptions::DISABLE_CLIENT_BYPASS) {
+		MutexHolder holder(lock);
+		ASSERT(!value.present() && !networkStartSetup);
+		disableBypass = true;
+		bypassMultiClientApi = false;
```

If this is already true, we should throw an `invalid_option` error.